.travis.yml

```diff
@@ -17,81 +17,57 @@
 # under the License
 #
 
+os: linux
+dist: focal
+language: cpp
+env:
+- PYTHON=python3
 jobs:
   include:
-  - os: linux
-    dist: xenial
-    language: cpp
-    compiler: gcc
-    env:
-    - OPENSSL_ia32cap='0x00000000'
-  - os: linux
-    dist: xenial
-    language: cpp
-    compiler: clang
-    env:
-    - OPENSSL_ia32cap='0x00000000'
+  - compiler: gcc
+  - compiler: clang
     # c-threaderciser test hangs on older clang
```

Comment:
> I was hoping to get Travis to build this PR but having force pushed a change it looks like it's not doing another build! :-(

Could be due to syntax error, you are missing `env:` here!

Also, I have my doubts about few other things in this file. There should be a travis syntax checker somewhere! https://github.com/travis-ci/travis.rb#lint maybe, but I hoped for something online, where I can paste file and it will give me report.

Reply:
Thanks for that syntax checking! It seems to have launched a travis job now.